doc/dev/development.md

```diff
@@ -140,11 +149,18 @@ git remote set-url --push upstream no_push
 
 ### Committing changes to your fork
 
-When you're happy with some changes, you can commit them and push them to your fork:
+When you're happy with some changes, you can commit them to your fork:
```

Comment:
My point is that in the sentence above, it's not correct to say "commit to your fork". You commit to your repo, and then you push to your fork.